src/FlaUI.Core/Application.cs

```diff
@@ -13,7 +13,7 @@ namespace FlaUI.Core
 {
     public class Application : IDisposable
     {
-        private static readonly ILogger Log = new ConsoleLogger();
+        public ILogger Log { get; }

```

Comment:
But the Log property from application is not used (or is it?). We always use the Logger.Default. ALso when setting the Log in Application, this would only affect the log messages written by the Application class and not from other classes.